Verbatim `util/unit_growth.v` section of patches/prosa-v06-rocq93-official-proof-closure-common.patch
(sha256 6819bfa9ac0bb9cfd6bfc7182d57e22463cf7a57b38ce9af89c8e7de96868050), split out for classic official closures.
--- a/util/unit_growth.v
+++ b/util/unit_growth.v
@@ -1,5 +1,6 @@
 From mathcomp Require Import ssreflect ssrbool eqtype ssrnat zify.zify.
 Require Import prosa.util.tactics prosa.util.notation prosa.util.rel.
+Set SsrOldRewriteGoalsOrder.
 
 (** We say that a function [f] is a unit growth function iff for any
     time instant [t] it holds that [f (t + 1) <= f t + 1]. *)
